### DISCUSSION TITLE
Detect if a user is smiling

### DISCUSSION THREAD

**Discussion by ChristopherGabba** [luicfrr/react-native-vision-camera-face-detector#68] [General]
Hello @luicfrr !

I have a unique use case where it would be awesome to be able to detect if a user is smiling on the camera. Any idea how I could execute this?

Thanks,
Chris

**Comment by luicfrr** [comment]
take a look into `MLKit` [documentation](https://developers.google.com/ml-kit/vision/face-detection/ios#1.-configure-the-face-detector)

**Reply by ChristopherGabba** [reply]
Thanks @luicfrr! I managed to get it working but Android has some issues. Can you take a look at this thread?

https://github.com/mrousavy/react-native-vision-camera/issues/3212
